subsys/bluetooth/host/att.c

```diff
@@ -588,7 +588,7 @@ static uint8_t att_handle_rsp(struct bt_att_chan *chan, void *pdu, uint16_t len,
 	k_delayed_work_cancel(&chan->timeout_work);
 
 	if (!chan->req) {
-		BT_WARN("No pending ATT request");
+		BT_DBG("No pending ATT request");
```

Comment:
Under normal situation there is something wrong when the remote respond when there is no request pending, so I wonder if the call path for security failing needs some changes in order not to trigger this.

Reply:
I considered this also:
```
static void bt_att_encrypt_change(struct bt_l2cap_chan *chan,
				  uint8_t hci_status)
...
	if (hci_status) {
		if (att_chan->req) {
			att_handle_rsp(att_chan, NULL, 0,
				       BT_ATT_ERR_AUTHENTICATION);
		}

		return;
	}
```
That would mean we wouldn't call `att_process` in this case, that should be fine, right?

Reply:
Yep, Im fine with doing the check for pending req.

Reply:
The check has been added.